hermes/views.py

```diff
@@ -395,14 +401,35 @@ def create(self, request, *args, **kwargs):
                     del data['files']
                 if 'file_comments' in data:
                     del data['file_comments']
-                message_uuid = submit_to_hop(request, data)
+                metadata = {'topic': data['topic']}
```

Comment:
Or did all the uuid logic get moved here?

Reply:
Yeah, I separated out the packing / generating the uuid from the submitting, so that we could have the uuid show up in the GCN submission, and have the gcn number show up in the final submitted hermes message too.